src/Presets/stubs/Tailwind/styles/components/_comments.scss

```diff
@@ -17,7 +17,7 @@
   @apply block appearance-none w-full py-1 px-2 mb-1 text-base leading-normal bg-white text-gray-700 border border-gray-500 rounded;
 
   &:focus {
-    @apply shadow-outline;
+    @apply ring;
```

Comment:
This class doesn't exist anymore - see the [Tailwind 2.0 upgrade guide section on shadow-outline](https://tailwindcss.com/docs/upgrading-to-v2#replace-shadow-outline-and-shadow-xs-with-ring-utilities) for more context.